apps/evm/src/clients/subgraph/__mocks__/index.ts

```diff
@@ -13,4 +15,6 @@ export const getBscProposal = vi.fn(async () => ({
 
 export const getNonBscProposals = vi.fn(async () => nonBscProposalsResponse);
 
+export const getBscCorePoolParticipantsCount = vi.fn(async () => bscCorePoolParticipantsCount);
+
 export const getIsolatedPoolParticipantsCount = vi.fn();
```

Comment:
I'd suggest mocking the calls made to fetch participant counts the same way (with the current approach, `getIsolatedPoolParticipantsCount` is mocked to returned `undefined` and its returned value is later mocked in the tests directly, while `getBscCorePoolParticipantsCount` is mocked to returned values).

It might be useful to dynamically mock the returned values based on the `__mocks__/api/pools.json` file content, since it's what we use as mock data throughout tests.